cob_monitoring/src/battery_monitor.py

```diff
@@ -74,9 +80,26 @@ def __init__(self):
             color_critical_param = [1.0, 0.0, 0.0, 1.0]
         self.color_critical = ColorRGBA(*color_critical_param)
 
-        self.interval_duration_warning = rospy.get_param("~interval_duration_warning", 30.0)
-        self.interval_duration_error = rospy.get_param("~interval_duration_error", 15.0)
-        self.interval_duration_critical = rospy.get_param("~interval_duration_critical", 5.0)
+
+        ### Set warning intervals ###
+        self.horn_interval_duration_warning = rospy.get_param("~horn_interval_duration_warning", 1200.0)    # default: 20 minutes
+        self.horn_interval_duration_error = rospy.get_param("~horn_interval_duration_error", 600.0)         # default: 10 minutes
+        self.horn_interval_duration_critical = rospy.get_param("~horn_interval_duration_critical", 300.0)   # default: 5 minutes
+
+        self.light_interval_duration_warning = rospy.get_param("~light_interval_duration_warning", 30.0)
+        self.light_interval_duration_error = rospy.get_param("~light_interval_duration_error", 15.0)
+        self.light_interval_duration_critical = rospy.get_param("~light_interval_duration_critical", 5.0)
```

Comment:
you changed the parameter name from `interval_duration_warning` to `light_interval_duration_warning` this makes it incompatible to existing config files